src/js/features/repoLookup/repoList/repoList.component.js

```diff
@@ -0,0 +1,25 @@
+export default class RepoListComponent extends React.Component {
```

Comment:
Specially with presentational components, it is good practice to define the propTypes (Interface) in `static propTypes = {}` and `static defaultProps = {}` if necessary so that you cover cases of, in this case, repoData being undefined (not passed as props)